tests/foreman/cli/test_contentviews.py

```diff
@@ -633,7 +633,7 @@ def test_cv_associate_components_composite_negative(self):
         with self.assertRaises(CLIFactoryError):
             result = make_content_view({
                 u'organization-id': self.org['id'],
-                u'component-ids': cv_version.stdout[0]['id']
+                u'component-ids': cv_version.stdout[0]['id'],
```

Comment:
I actually like this change. I think adding trailing commas to the end of each line in a list of arguments is good because:

* It allows you to add arguments without the risk of forgetting to add a trailing comma to the previous line, and
* it allows you to rearrange arguments without fiddling around with commas.